homeassistant/components/axis/binary_sensor.py

```diff
@@ -35,16 +36,19 @@ def __init__(self, event, device):
         """Initialize the Axis binary sensor."""
         self.event = event
         self.device = device
-        self.delay = device.config_entry.options[CONF_TRIGGER_TIME]
         self.remove_timer = None
+        self.unsub_dispatcher = None
 
     async def async_added_to_hass(self):
         """Subscribe sensors events."""
         self.event.register_callback(self.update_callback)
+        self.unsub_dispatcher = async_dispatcher_connect(
+            self.hass, AXIS_REACHABLE, self.update_callback)
 
-    def update_callback(self):
+    def update_callback(self, delay=None):
```

Comment:
Will this callback be called in the event loop by the library? Then we should annotate it with `@callback` so that home assistant also runs in in the event loop when the dispatcher triggers it.

Reply:
And change `schedule_update_ha_state` to `async_schedule_update_ha_state` inside.